shopinvader/services/customer.py

```diff
@@ -193,3 +199,13 @@ def _prepare_create_response(self, binding):
             "role": binding.role,
         }
         return response
+
+    def _valid_before_binding(self, params):
+        fields = ["partner_email", "cnpj_cpf"]
+        values = [params.get("email"), params.get("cnpj_cpf")]
```

Comment:
what's 'cnpj_cpf` ????

Reply:
I removed the checks from this field and transferred it to a more suitable module.